app/assets/javascripts/pages/members.coffee

```diff
@@ -20,6 +20,7 @@ form_signup.submit((e)->
       dataType: 'json'
       data: { website_member: { token: gon.token, email: form_signup.find('#email').val() } }
       success: (data) ->
+        alert 'Confirm email first'
```

Comment:
nu imi place ca folosesti alerturi. Nu asta e idea. Incearca sa faci un div in care sa pui clasele de bootstrap si sa adaugi cu jquery mesajele astea in pagina nu cu alerturi...